hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java

```diff
@@ -106,6 +106,13 @@ protected UserVolumeInfo addVolumeToOwnerList(UserVolumeInfo volumeList,
       objectID = volumeList.getObjectID();
     }
 
+    // Sanity check, a user should not own same volume twice
+    //  TODO: May want to remove this due to perf if user owns a lot of volumes.
+    if (prevVolList.contains(volume)) {
```

Comment:
I am thinking instead of returning an error like AccessDenied, can we return a code that this owner is already owner for this volume.
Because access Denied looks not proper here.

Reply:
I was also thinking of adding a new type of exception. But now I think @dineshchitlangia 's suggestion might be better -- handle it silently and maybe throw a warning.

Reply:
Can we document this behavior, so that the end-user knows about this behavior?

Reply:
IMO the client shouldn't notice this. The warnings should only show up on the server (if any).